### PR TITLE
Multilingual quirk can no longer give you centcom encrypted language and costs 2 points instead of 3

### PR DESCRIPTION
**Pull request by SapphicOverload**
# Document the changes in your pull request

I forgor

# Changelog

:cl:   
tweak: no more learning CC encrypted language with multilingual quirk
tweak: multilingual costs 2 points instead of 3
/:cl:

### CODE REVIEW

File: code/datums/traits/good.dm [hunk 1]
```diff
@@ -373,6 +373,10 @@
 	var/datum/language/specific
 	gain_text = span_notice("You have learned to understand an additional language.")
 	lose_text = span_notice("You have forgotten how to understand a language.")
+	var/list/blacklisted_languages = list(	/datum/language/codespeak,
+											/datum/language/narsie,
```
Comment (ThatLing) [inline code/datums/traits/good.dm]: Just use spaces for indentation

**Comment by Hislyna** [comment]
This is a 3 point quirk (sheltered is -2 for ref) and curator has this language. I see no reason for a chance to get this language when it occurs randomly and rarely, especially at the cost

**Comment by iamthesilly** [comment]
changed it to 2 points

**Comment by MajManatee** [comment]
If curator has it, then you should be able to roll it. That sounds fair.

**Review by MajManatee — APPROVED** [review]
i guess its fine then...?

**Pull request merged**